java/custom/org/mozilla/Test.java

```diff
@@ -0,0 +1,6 @@
+package org.mozilla;
```

Comment:
Can you move this class in the bench/ directory? This way we don't need to package it in classes.jar for release.